Mathlib/LinearAlgebra/UnitaryGroup.lean

```diff
@@ -79,6 +79,24 @@ theorem det_of_mem_unitary {A : Matrix n n α} (hA : A ∈ Matrix.unitaryGroup n
   · simpa [star, det_transpose] using congr_arg det hA.1
   · simpa [star, det_transpose] using congr_arg det hA.2
 
+open scoped Kronecker in
+/-- The kronecker product of two unitary matrices is unitary. -/
```

Comment:
```suggestion
/-- The kronecker product of two unitary matrices is unitary.

This is stated for `unitary` instead of `unitaryGroup` as it hold even for non-commutive coefficients. -/
```